templates/openstack/index.html

```diff
@@ -24,346 +24,691 @@ <h1>OpenStack, delivered</h1>
-      <li class="p-list__item is-ticked">When your team is ready, we hand over the keys</li>
-    </ul>
-    <p>
-      <a class="p-button--positive" href="/openstack/managed">
-        Learn about Managed OpenStack
-      </a>
-    </p>
+<section class="p-strip">
+  <div class="row u-vertically-center">
+    <div class="col-6 u-hide--small u-align--center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/ba7bf5ff-Openstack-cloud-optimisation.svg",
+        alt="",
+        width="390",
+        height="174",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+    <div class="col-6">
+      <h2>Maximum multi-cloud cost optimisation</h2>
+      <h3 class="p-heading--4">Run your workloads where it makes the most sense from an economical point of view</h3>
+      <p>The future of infrastructure is multi-cloud.</p>
+      <p>Using private and public cloud infrastructure at the same time allows you to optimise your CapEx and OpEx costs.</p>
+      <button class="p-link--external p-button--neutral" href="https://assets.ubuntu.com/v1/d4766546-Private-Cloud-Build_Datasheet.pdf?_ga=2.41518498.1083732777.1610363777-647098137.1594061070">Download datasheet</button>
+    </div>
   </div>
 </section>
 
-<section class="p-strip is-bordered">
-  <div class="u-fixed-width">
-    <h2>Choose your OpenStack package</h2>
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-6">
+      <h2>Cheaper VMs</h2>
+      <h3 class="p-heading--4">Machines cost less if they run in a private cloud</h3>
+      <p>Compare the prices of sample VMs running on a baseline Charmed OpenStack cloud over a three-year period. The calculations include hardware costs, typical hosting charges and <a href="/openstack/managed">fully-managed service</a>*.</p>
+      <a class="u-sv2" href="/openstack/tco-calculator">Check pricing for other VMs&nbsp;&rsaquo;</a>
+      <p>* Exact prices may vary depending on the used hardware, cloud configuration, local hosting charges and salary costs.</p>
+    </div>
+    <div class="col-6 u-hide--small u-align--center">
+      <table aria-label="Comparison table for pricing of VMs">
+        <thead>
+          <tr>
+            <th>VM flavour</th>
+            <th class="u-align--right">Number of VMs</th>
+            <th class="u-align--right">Price per hour</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <aside class="p-accordion">
+                <ul class="p-accordion__list">
+                  <li class="p-accordion__group">
+                    <div role="heading" aria-level="3" class="p-accordion__heading">
+                      <button type="button" class="p-accordion__tab" id="tab1" aria-controls="tab1-section" aria-expanded="false">c1.small</button>
+                    </div>
+                    <section class="p-accordion__panel" id="tab1-section" aria-hidden="true" aria-labelledby="tab1">
```

Comment:
I think we need to override the padding in this section. Could you add `style:"padding-left: 1rem;"`

Reply:
On all the panels